### ISSUE TITLE
Page Layouts A11Y

### ISSUE THREAD

**Issue by BrittanyIRL** [GoogleForCreators/web-stories-wp#5876]
## Feature Description
Now that page layouts is fleshed out we need to make sure it is also keyboard accessible and has proper roles for Screen Reader users. 

## Additional Context

---

_Do not alter or remove anything below. The following sections will be managed by moderators only._

## Acceptance Criteria
- ~Page Layouts in the panel (previews) should be buttons to show that they are interact-able. Make sure that these buttons announce themselves properly with screen readers and that keying 'enter' uses the page layout.~ (Updated in #5890 ) 
- Page Layouts in panel should be keyboard navigable  w/ arrows and tabs (see images).

**Comment by BrittanyIRL** [comment]
Testing copied from PR: 

- Should be able to navigate page layouts with arrow keys
- Tabbing from a page layout will send you to the canvas
- 'Enter' when hit from a page layout should either swap the page layout if the canvas is empty or open a dialog. If a dialog enters you should be able to tab to the buttons on it. When you close the dialog the focus should return to the page layout you were on when you entered the dialog.
- Confirm the same holds true for RTL
- Confirm screen reader still gives accurate information about page layouts
- Confirm scroll is still good to go

**Comment by BrittanyIRL** [comment]
hey @csossi - just curious when you think you'll get to this? It's been ready for qa for 8 days.

**Comment by csossi** [comment]
Editor is crashing on loading PR 5989 to test

![image.png](https://images.zenhubusercontent.com/5ea085e4704e72c0a99c6997/3d245f63-cfc8-4a21-8f8b-6d7924d1f118)

**Comment by BrittanyIRL** [comment]
> Editor is crashing on loading PR 5989 to test 

Oh the joys of a very old PR getting tested. Looking into it now.

**Comment by BrittanyIRL** [comment]
ok, should be fixed!

**Comment by csossi** [comment]
Verified in QA
